Scripts/quicktest.out

```diff
@@ -9,19 +10,40 @@ Compiling on C#
 
 Dafny program verifier finished with 0 verified, 0 errors
 (42, 131)
-Compiling to Javascript
+Compiling on Javascript
```

Comment:
This is a somewhat unfortunate change in wording, IMO.

Reply:
I'm not sure where my brain was -- I agree.